### PR TITLE
refactor: remove code specific to IE 9 and IE 10 support

### PR DESCRIPTION
**Pull request by AndrewKushnir**
This PR contains several commits that remove code specific to IE 9 and IE 10 support.
The code is no longer needed since IE 9 and IE 10 support is removed.

### CODE REVIEW

File: packages/platform-browser/animations/test/BUILD.bazel [hunk 1]
```diff
@@ -39,37 +39,6 @@ jasmine_node_test(
 
 karma_web_test_suite(
     name = "test_web",
-    tags = [
-        # FIXME: fix on saucelabs
```
Comment (IgorMinar) [inline packages/platform-browser/animations/test/BUILD.bazel:43]: yay! :-)

**Review by IgorMinar — APPROVED** [review]
I'd expect there to be more stuff to remove from forms, but it might be hard to identify the code paths if they were not well documented.

Otherwise this looks great! Thank you!

Reviewed-for: global-approvers

File: packages/router/test/bootstrap.spec.ts [hunk 2]
```diff
@@ -301,7 +301,7 @@ describe('bootstrap', () => {
     await router.navigateByUrl('/aa');
     window.scrollTo(0, 5000);
 
-    // IE 9/10/11 use non-standard pageYOffset instead of scrollY
+    // IE 11 use non-standard pageYOffset instead of scrollY
```
Comment (gkalpak) [inline packages/router/test/bootstrap.spec.ts:304]:
```suggestion
    // IE 11 uses non-standard pageYOffset instead of scrollY
```

**Review by gkalpak — APPROVED** [review]
Yay! :shipit: 

Reviewed-for: docs-infra, fw-elements, fw-upgrade

**Review by jelbourn — APPROVED** [review]
LGTM

Reviewed-for: public-api

**Comment by AndrewKushnir** [comment]
[Presubmit](http://test/OCL:336813608:BASE:336813624:1602567969624:a209796d).

**Comment by AndrewKushnir** [comment]
Merge assistance: it looks like there is a GitHub UI glitch where some reviewers were added twice and I'm unable to remove them from the list (thus the `ci/angular: merge status` is failing).

**Pull request closed without merging**